appcli/cli_builder.py

```diff
@@ -157,26 +175,16 @@ def check_environment():
                 "Cannot run without all mandatory environment variables defined")
             sys.exit(1)
 
-    # --------------------------------------------------------------------------
-    # CREATE_CLI: LOGIC
-    # --------------------------------------------------------------------------
-
-    install_command = InstallCli(configuration).command
-    cli.add_command(install_command)
-
-    main_commands = MainCli(configuration).commands
-    for command in main_commands:
+    for command in all_commands.values():
         cli.add_command(command)
 
-    configure_command = ConfigureCli(configuration).command
-    cli.add_command(configure_command)
+    for command in configuration.subcommands:
```

Comment:
Like the above, let make this clearer by calling it `configuration.custom_commands` so it's clear these are user defined